alexandria/core/serializers.py

```diff
@@ -183,7 +186,9 @@ def get_webdav_url(self, instance):
             return None
         request = self.context.get("request")
         host = request.get_host() if request else "localhost"
-        user, group = get_user_and_group(request)
+        user, group = import_string(settings.ALEXANDRIA_GET_USER_AND_GROUP_FUNCTION)(
+            self.context["request"]
+        )
```

Comment:
I think this could be factored out into a method here - same (relatively-complex) code pattern 3 times in the same class..

Reply:
Makes sense, done